xbmc/cores/paplayer/PAPlayer.cpp

```diff
@@ -374,6 +387,23 @@ bool PAPlayer::QueueNextFileEx(const CFileItem &file, bool fadeIn)
   if (si->m_endOffset)
     streamTotalTime = si->m_endOffset - si->m_startOffset;
 
+  // Seek to a resume point
+  if (si->m_fileItem.HasProperty("StartPercent") &&
+      (si->m_fileItem.GetProperty("StartPercent").asDouble() > 0) &&
+      (si->m_fileItem.GetProperty("StartPercent").asDouble() <= 100))
+  {
+    si->m_seekFrame =
+        si->m_audioFormat.m_sampleRate *
+        CUtil::ConvertMilliSecsToSecs(static_cast<int>(+(static_cast<double>(
```

Comment:
As a question to learn something new, what's the background of the `"+"` there. Is new to me and don't quite understand what it does :sweat_smile:?

Reply:
An interesting typo, no idea how that got in there and then I am blind to it. At least it does no harm.